spec/vcr_cassettes/manageiq/providers/ibm_cloud/vpc/cloud_manager/refresher.yml

```diff
@@ -32,18 +32,20 @@ http_interactions:
+      string: '{"access_token":"eyJraWQiOiIyMDIwMDkyMjE4MzMiLCJhbGciOiJSUzI1NiJ9.eyJpYW1faWQiOiJJQk1pZC0wNjAwMDA0UzM4IiwiaWQiOiJJQk1pZC0wNjAwMDA0UzM4IiwicmVhbG1pZCI6IklCTWlkIiwianRpIjoiMmFiYmE5YTEtNjliNS00ZWNhLTg2NjYtNWU5YWUzNTZkMjg5IiwiaWRlbnRpZmllciI6IjA2MDAwMDRTMzgiLCJnaXZlbl9uYW1lIjoiQnJhZCIsImZhbWlseV9uYW1lIjoiQmxhbmNldHQiLCJuYW1lIjoiQnJhZCBCbGFuY2V0dCIsImVtYWlsIjoiYmxhbmNldHRAdXMuaWJtLmNvbSIsInN1YiI6ImJsYW5jZXR0QHVzLmlibS5jb20iLCJhY2NvdW50Ijp7InZhbGlkIjp0cnVlLCJic3MiOiJjNTZjOWEyNjhkMjNlMWIzMzlhYzE0Nzc0MzU4MTMzYyIsImltc191c2VyX2lkIjoiNzI4MjEwMyIsImZyb3plbiI6dHJ1ZSwiaW1zIjoiMTE2MDQ0NyJ9LCJpYXQiOjE2MDIxNzkwMTcsImV4cCI6MTYwMjE4MjYxNywiaXNzIjoiaHR0cHM6Ly9pYW0uY2xvdWQuaWJtLmNvbS9pZGVudGl0eSIsImdyYW50X3R5cGUiOiJ1cm46aWJtOnBhcmFtczpvYXV0aDpncmFudC10eXBlOmFwaWtleSIsInNjb3BlIjoiaWJtIG9wZW5pZCIsImNsaWVudF9pZCI6ImRlZmF1bHQiLCJhY3IiOjEsImFtciI6WyJwd2QiXX0.TSdCG3zWBkFNDMkbDePWsEdJboouuAtetA-vvU3jipnpf0i51LVdp-8Muj6q0PltSzNGAsLpSaN5KWSFbP1O7jNE1AHlCKpN_lcD30RIm74tyH-eFvIk33ktJSvCb77D21iUQ-FUwM33_qq7CXKDtqsfLJirZX9LD0gzeXEunJL_b7jnqdihV1hqnw-hrK6Ui4bRZXfGtcRWmU6W0sbUs38snL1ewP_1wnxuu2JybWAikZxzTUqWZpaz3jc7bSRkZ6CDSWTeV6az3kMX1y6qR1P0wdtmjuICv5vykqX4ZMVHrZurkWdL9FHs-aEHMLxuyvzqO6BV1d8-BqhMNT3cvg","refresh_token":"OKCmsuS5N0DWt_8D5OE75H1qZURVgEowgBTdAtODoP91lNwBwexQMFNSjjg0xM6TuxKgaBUbv-xD0OCg2iaqCYQyGoWrO0tshNDXm4XHizJBJY-h9_I5PHUN7gSWUhzmeDvJO3tOi6x6ePJg9rcVZw6omJ-sOuCwi1VvCkplI01A_5fXBkbEVQG6i3AR5pSXURDuIgUgaNgTkvq6nW1-QmX2fUwB7HAMmG-e3IyYbTVnNRlIqhy8_-XpLEDb4wwHp9Ran5gL0_-jEOj3LCUI1zOGHhz9IgeV-jaQVAWp5bZ9vYrroEOMTcjKqYmo_1JGZtyhk9e2SSB5c89-i4ggoMjM3cXOhJgR7wQNHVCA8xCSzAKrcV5Qr86Tqk7GHiCPLslS8eT_6xW_MyU8ipin78drrdNUXd4vg-ir3pnLPwXwxC_Up0f07sIUgaBr-GFoQaBmCl3s21NPE-4AWC32esdZiMNWmXOmLfrwgBCWalQZ6i-1Sx3eKrmAtWVtPcYFYJ2TtaVwG83znBX0HidAw8oZv4iDhC-BwxGOmTS8lIsSD_6Fw2H7TIqN7Z0ie0vkGijRifOHAV0RyFOE4ZPSP5hay_UHZ4qh8O-JtTNmxPuOqXy0ypgu4T6f4RMn5Exd4eusIix79r82FUS0Ejo5YKKND6_G54pybyK228pdrso9_JcEYzMkUjrA2I5lvsOb2EWk_r16MOnvVN18mH_8RatrBlmBB3HZBivHh9S969JTAWmVTcgvcghKLq_yn9rjisGmrhyqH_tUrgRnrUrfXccuerLkkg5Ljql7uEhHELLIu_tgX1Cv_2Zwzu4dLJdD4cLqz1kcz74NgMvh12yoa078rOIgrfd3_U-F7T9u32x7ggiyXpAmRoi8FK6SevY4ArTDl4qYepjgyth_1ZOx44lEZvhtahhm9hHlGvrr0dxokcnhCJKjfFrbQrAFBCT9uDqzK4-VwXZi7TCCSWAUr6psAUaMtljVbpgQiXTne_tzJi0OUvVHiUJxJOyPsjChmibAk6Be6trE4kImt5ORjTEgZvr4G2-pCISgIwZG05hIdKYiE9dpXHqWLg_So1AZ5ybUA_SgkFZ_6QcpOWnDTB7YmnJlczWb2TnvG_4Oo8pRrZ59SASzZXHtKDfb0QswS9c","ims_user_id":7282103,"token_type":"Bearer","expires_in":3600,"expiration":1602182617,"refresh_token_expiration":1604771017,"scope":"ibm
```

Comment:
So this is going to start failing 1 hour after this was recorded, I had been manually editing the expiration but this is obviously terrible :laughing: 

I opened a PR https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/81 so that this will be done automatically

Reply:
Wow that is painful - ok will make sure to update till 81 is merged.

Reply:
Okay @blancett go ahead and re-record this, should be good to go